package.json

```diff
@@ -4,9 +4,16 @@
   "description": "e-commerce",
   "main": "src/server.ts",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
```

Comment:
add "--coverage" we need that

Reply:
@Ke-bean

Reply:
Can you work on these changes requested by your peers @Ke-bean